README.md

```diff
@@ -34,6 +34,13 @@ Then select the wanted Terraform version to use with `chtf`.
 
     chtf 0.11.3
 
+If you wold like to default to a particular version so terraform when opening a new terminal you can add
+```
+chtf 0.11.14
```

Comment:
This would actually have to happen in lines 21 (before fi) and 30 (before end) in README, because `chtf` command is probably not recognised unless we have `/usr/local/share/chtf/chtf.*sh` in our filesystem and we've performed `source` command for it.

Reply:
@jylitalo updated PR as this is a suggestion and I assumed the user would knot it needs to go after a sourcing statement for path/to/binary.  Has kept me from having to select a version when opening a new terminal.  Thanks again for the tool. Makes it really easy to jump back and forth while upgrading TF.